### PR TITLE
Check server version

### PR DESCRIPTION
**Pull request by lego963**
If server is running it will be checked for version

### CODE REVIEW

File: too_simple_server/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+  register: CURRENT_VERSION
+  ignore_errors: yes
+
+- become: yes
+  block:
+    - name: Stop server
+      include_tasks: stop_server.yml
+    - name: Start server
+      include_tasks: start_server.yml
+  when: CURRENT_VERSION.stdout_lines[0] == LATEST_VERSION.stdout_lines[0] and CURRENT_VERSION.rc == 0
```
Comment (outcatcher) [inline too_simple_server/tasks/main.yml]: `CURRENT_VERSION.stdout_lines[0] == LATEST_VERSION.stdout_lines[0]`
Those two should be better saved as variables, like `current_version` and `latest_version`

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: too_simple_server/tasks/main.yml [hunk 2]
```diff
@@ -32,32 +49,10 @@
...
-    - name: Start server
-      command: "{{ server }} --config {{ server_config }}"
-      args:
-        creates: "{{ server_pid_file }}"
-    - name: Wait for port starts
-      wait_for:
-        port: "{{ server_port }}"
-        timeout: 60
+    - include_tasks: stop_server.yml
+      when: server_state == 'absent' or force_new_download
```
Comment (outcatcher) [inline too_simple_server/tasks/main.yml]: We should restart server anyway

File: too_simple_server/vars/main.yml [hunk 3]
```diff
@@ -1,10 +1,12 @@
-version: "0.4.3"
+current_version: "{{ CURRENT_VERSION.stdout_lines[0] }}"
+current_version_rc: "{{ CURRENT_VERSION.rc }}"
+latest_version: "{{ LATEST_VERSION.stdout_lines[0] | default('0.4.3') }}"
```
Comment (anton-sidelnikov) [inline too_simple_server/vars/main.yml]: default work here only if  default(some_variable, true)

**Review by anton-sidelnikov — APPROVED** [review]

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]
need fix

**Review by Polina-Gubina — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**